src/page-modules/assistant/trip/utils.ts

```diff
@@ -0,0 +1,256 @@
+import {
+  formatToClock,
+  secondsBetween,
+  secondsToDuration,
+} from '@atb/utils/date';
+import {
+  Leg,
+  TripPattern,
+} from '@atb/page-modules/assistant/server/journey-planner/validators';
+import { Language, TranslateFunction, PageText } from '@atb/translations';
+import dictionary from '@atb/translations/dictionary';
+import { screenReaderPause } from '@atb/components/typography/utils';
+import { getQuayName } from '@atb/page-modules/assistant/trip/trip-pattern-header';
```

Comment:
Relative paths for the imports of `Leg, TripPattern` and `getQuayName` as we discussed on Slack 👍